### PR TITLE
Interactive todo list

### PR DESCRIPTION
**Pull request by themonster2015**
In this PR, I've added the toggle todo function to the to-do list I created yesterday. 

- Added a new JS file for managing the toggling and importing it as a module.
- Added event listener to the checkbox to toggle todo.
- Store the updated todo array in local storage so the user gets the correct list values after the page reloads.
- Page reloads when the user toggles a to-do item.

### CODE REVIEW

File: src/statusChange.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  const newtodos = todos.map((e) => {
+    if (e.index === Number(id)) {
+      e.completed = !e.completed;
+      return e;
+    }
+    return e;
+  });
+  window.localStorage.setItem('todos', JSON.stringify(newtodos));
+  // eslint-disable-next-line
+  location.reload();
```
Comment (bellom) [inline src/statusChange.js]: So with these pieces of code, once the user checked an item, the page seems to reload, that is not what we wanted right.

we want a user to check any item and our localStorage should save the changes.

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @themonster2015  👏 , 

### Awesome job so far!! :+1: 

#### You have done wonderfully well setting up linter and good commit messages 🔥, However, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Required Changes! ♻️ 

_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**It would be nice to implement a method to drag-and-drop sorting for this project, don't forget to use modules.**


### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/statusChange.js (hunk 1 above)
Comment (themonster2015) [reply src/statusChange.js]: Hi,

Thanks for your comment. I updated localstorage with the line:

  window.localStorage.setItem('todos', JSON.stringify(newtodos));
   
The location.reload() part was just for better user experience.

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hi there, 

Awesome job on making the changes requested by the previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. Let's work on those.


 ## Required Changes ♻️
_Kindly check the comments under the review._


Happy coding, Cheers!🚀🚀

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it's valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 2]
```diff
@@ -16,23 +18,37 @@ const todos = [{
...
     const todo = document.createElement('div');
-    todo.setAttribute('class', 'todoItem');
+    todo.setAttribute('id', `${el.index}`);
+    todo.classList.add('todoItem');
     const titleCheckbox = document.createElement('div');
+
+    const checkbox = document.createElement('input');
+    checkbox.type = 'checkbox';
+    checkbox.addEventListener('click', (e) => {
+      const id = e.target.parentNode.parentNode.getAttribute('id');
```
Comment (mikenath223) [inline src/index.js:42]: - [x] Kindly ensure that when I click to complete a to-do the completed state for that particular to-do actually gets saved in the localstorage. So click a to do item and refresh the page and the to do checkbox should still be checked for that item.

**Comment by mikenath223** [comment]
- [ ] OPTIONAL - Kindly ensure you update the readme.md following this [template](https://github.com/microverseinc/readme-template).

File: src/index.js (hunk 2 above)
Comment (mikenath223) [reply src/index.js:42]: - Hi, could you make it so when the user clicks the checkbox and refreshes the page the checkbox **still remains checked**.

Comment (themonster2015) [reply src/index.js:42]: Hi. Thanks for clarifying. Now I see what you wanted. I've fixed it so that the checkbox remains checked for completed items.

Comment (rloterh) [reply src/index.js:42]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Yen, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾 🍾 :shipit: .
Congratulations! 🎉 🎉 🥂


<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have no more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SpaYco — COMMENTED** [review]
Hello 👋,

_This is an additional review as a result of the second opinion request._

It looks like everything got resolved here, so I have nothing else to say other than good luck with your future projects!

Keep rocking 🚀!

**Pull request merged**